module2/homework/maxOfVector/maxOfVector.hpp

```diff
@@ -3,6 +3,13 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    // Would fail for empty vec - but the behaviour is unspecified in task
+    auto currentMax = *vec.begin();
```

Comment:
I don't understand why u used pointer. Could u explain?

Reply:
> 
> 
> I don't understand why u used pointer. Could u explain?

That's not a pointer.
Vec.begin() returns an iterator.
Iterators overload '*' operator, so that it returns the value the iterator is currently at.